flink-tests/src/test/java/org/apache/flink/test/checkpointing/RescalingITCase.java

```diff
@@ -852,6 +862,7 @@ public void invoke(IN value) throws Exception {
 
         private static final long serialVersionUID = 7512206069681177940L;
         private static volatile CountDownLatch workStartedLatch = new CountDownLatch(1);
+        private static volatile CountDownLatch canFinishLatch = new CountDownLatch(0);
```

Comment:
why are the latches volatile?

Reply:
Good question... I think the answer is it is not necessary here. I had not thought about it, but just copied over the other latch :sweat_smile: 

I will remove the volatile from both of the latches.